### PR TITLE
bh - fixed RecommendationController, removed line for id param in POST

### PR DESCRIPTION
**Pull request by bhuang01**

### CODE REVIEW

**Review by peterlototsky — APPROVED** [review]
LGTM

File: src/test/java/edu/ucsb/cs156/example/controllers/RecommendationControllerTests.java [hunk 1]
```diff
@@ -168,7 +168,6 @@ public void an_admin_user_can_post_a_new_recommendation() throws Exception {
             LocalDateTime ldt1 = LocalDateTime.parse("2022-01-03T00:00:00");
 
             Recommendation one = Recommendation.builder()
-                            .id(1)
```
Comment (coding-me-softly) [inline src/test/java/edu/ucsb/cs156/example/controllers/RecommendationControllerTests.java:171]: There are several other places where the `.id` method is called on the Recommendation builder.  Do those all have to change?

**Review by coding-me-softly — CHANGES_REQUESTED** [review]

File: src/test/java/edu/ucsb/cs156/example/controllers/RecommendationControllerTests.java (hunk 1 above)
Comment (bhuang01) [reply src/test/java/edu/ucsb/cs156/example/controllers/RecommendationControllerTests.java:171]: > There are several other places where the `.id` method is called on the Recommendation builder. Do those all have to change?

image

Comment (bhuang01) [reply src/test/java/edu/ucsb/cs156/example/controllers/RecommendationControllerTests.java:171]: Just for POST

**Review by sophiacheng01 — APPROVED** [review]
LGTM

**Pull request merged**